pkg/testing/fixture.go

```diff
@@ -1243,7 +1237,7 @@ type AgentStatusOutput struct {
 }
 
 func (aso *AgentStatusOutput) IsZero() bool {
-	return aso.Info.ID == ""
+	return aso.Info.ID == "" && aso.Message == "" && aso.Info.Version == ""
```

Comment:
This makes more sense actually. As `IsZero` should be if nothing is outputted at all.